content/en/docs/concepts/signals/metrics.md

```diff
@@ -73,6 +73,33 @@ The instrument kind is one of the following:
   A histogram is a good choice if you are interested in value statistics. For
   example: How many requests take fewer than 1s?
 
+### Synchronous and asynchronous instruments
+
+Asynchronous instruments are useful in several circumstances, such as:
+
+- When updating a counter is not computationally cheap, and you don't want the
+  current executing thread to wait for the measurement
+- Observations need to happen at frequencies unrelated to program execution
+  (i.e., they cannot be accurately measured when tied to a request lifecycle)
+- There is no known timestamp for a measurement value
+
+In cases like these, it's often better to observe a cumulative value directly,
+rather than aggregate a series of deltas in post-processing (the synchronous
+example). Take note of the use of `observe` rather than `add` in the appropriate
+code examples below.
```

Comment:
```suggestion
In the previous cases, it's often better to observe a cumulative value directly
rather than aggregate a series of deltas in post-processing (the synchronous
example). Note here the use of `observe` instead of `add` in the following
examples.
```

Reply:
Which examples?

Reply:
I would wager few outside of those who contribute to OTel would understand this sentence:

> In cases like these, it's often better to observe a cumulative value directly,
rather than aggregate a series of deltas in post-processing (the synchronous
example).

Is it really needed?

Reply:
> Which examples?

the JS examples, so that sentence makes no sense anymore

Reply:
```suggestion
```

Should I remove that entirely, wdyt?

Reply:
I removed a lot of that content and point to the guidelines from the spec which have all that we need.